docs/guide.rst

```diff
@@ -830,6 +830,40 @@ copied to the guest.
     ``finish`` steps yet.
 
 
+.. _execute-tests:
+
+Execute Tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+TODO: Here goes content of examples/run:
+https://tmt.readthedocs.io/en/stable/examples.html#run
+
+Rerunning failed tests of previously executed runs
+------------------------------------------------------------------
+
+Executing failed tests again after fixing them is now possible
+with `tmt run --all --again tests --failed-only`.
```

Comment:
How about adding `--last` in the example? Otherwise it's not possible to copy&paste. Sure, you mention that below but it will be easier if the example was a working one already.